string id_pareto_front
int32 num_ind
---
float64[] var_space